source/api_documentation/network_integration/advertiser_campaigns/index.rst

```diff
@@ -294,12 +292,43 @@ Node Parameters and Usage
 
   * - VerifyLocation
     - prompt
-    - Prompt the caller to verify the guessed location or confirm through input. Useful if geographical data is important or useful in a condition node.
+    - Prompts the caller to verify the guessed location or confirm through input. Useful if geographical data is important or useful in a condition node.
 
   * - DynamicRoute (beta - read only)
     - \*dynamic_route_destination
     - Forwards the call to a destination that is extracted from a custom data field specified in dynamic_route_destination. The destination must be a phone number or if you are SIP integrated, can be a string that is routable by your SIP infrastructure.
 
+  * - AnyKeyPress
+    - \*prompt
+
+    - Allows the caller to make any keypress to continue the call.
```

Comment:
For all the node types that require a prompt, using `Prompts the caller` instead of `Allows the caller` feels like more of a "call to action" since the ideal scenario for most calls is that the caller does respond in some way.

What do you think?

Reply:
Yeah, I went back and forth between the two because I couldn't decide. I'll go with `Prompts the caller` since you gave me your buy-in!